tests/views.py

```diff
@@ -0,0 +1,38 @@
+from django.shortcuts import render
+from notes.models import Record
+import random
+
+
+# NOT FINISHED
+# Create your views here.
+def test(request, pk_topic, pk_note):
+    profile = request.user.profile
+    topic = profile.topic_set.get(id=pk_topic)
+    note = topic.note_set.get(id=pk_note)
+    records = note.record_set.all()
+
+    records_ids = Record.objects.filter(note=note).values_list('id', flat=True)\
```

Comment:
Tutaj najprościej będzie jednak tak:
1. wyrzucić z records_ids choice_id (metoda exclude)
2. posortować losowo (order_by['?']), wyciągnąć trzy pierwsze wpisy (order_by('?')[:3]) i wynik przypisać do incorrect_answers.

poza zwięzłością zapisu (całość w dwóch linijkach), dodatkowo zyskujemy próbkowanie w bazie, a nie w pythonie (miałoby to ogromne znaczenie gdyby wpisów, z których losujemy było dużo).